test/Microsoft.AspNet.Mvc.Core.Test/Formatters/StreamOutputFormatterTest.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.AspNet.Mvc;
+using Microsoft.Net.Http.Headers;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc
+{
+    public class StreamOutputFormatterTest
+    {
+        [InlineData(typeof(Stream), typeof(FileStream), "text/plain", "text/plain")]
+        [InlineData(typeof(object), typeof(FileStream), "text/plain", "text/plain")]
+        [InlineData(typeof(object), typeof(MemoryStream), "text/plain", "text/plain")]
+        [InlineData(typeof(object), typeof(object), "text/plain", null)]
+        [InlineData(typeof(object), typeof(string), "text/plain", null)]
+        [InlineData(typeof(object), null, "text/plain", null)]
+        [InlineData(typeof(IActionResult), null, "text/plain", null)]
+        [InlineData(typeof(IActionResult), typeof(IActionResult), "text/plain", null)]
+        [Theory]
+        public void GetSupportedContentTypes_ReturnsAppropriateValues(Type declaredType, Type runtimeType,
+                                                                      string contentType, string expected)
```

Comment:
:+1: to a new rule as long as it's got the verbiage @rynowak usually includes e.g. "if wrapping parameters / arguments over multiple lines, give them all their own lines."  note this doesn't rule out either placing everything on one line when it fits or breaking a single long (likely delegate) argument over multiple lines where necessary.